lib/pizza.rb

```diff
@@ -0,0 +1,28 @@
+require './lib/prices'
+require 'pry'
+class Pizza
+attr_reader :price
+
+  include Prices
+
+  def initialize(size, type, crust)
+    @size = size
+    @type = type
+    @crust = crust
+    @price = 0
```

Comment:
I am going to finish this over the weekend, I was excited that I improved from the last time I did this problem. The difference from last mod is, I am **excited** to tackle the problem over the weekend as I know I can get it with some more time. = )